src/utils/Chart.js

```diff
@@ -0,0 +1,35 @@
+const d3 = require('d3');
+
+const Chart = {
+  getDataMinMaxValues (data, axis) {
+    const max = d3.max(data, d => {
+      return Math.ceil(d[axis] / 1000) * 1000;
```

Comment:
Not sure why d3 works that way. It just sounds really strange to divide a number by 1000 and then multiplying it by 1000 gives a different result. 

I know Math.ceil returns an integer. And you are trying to get a smaller number by diving it by 1000. But anytime you have a number less than 1, `Math.ceil` always returns 1. How does that change anything? Can you give me an example? Sorry, I'm just curious to know.